lib/storage/data/file/DataFileStore.js

```diff
@@ -45,13 +45,20 @@ class DataFileStore {
      *   sync calls that ensure files and directories are fully
      *   written on the physical drive before returning an
      *   answer. Used to speed up unit tests, may have other uses.
+     * @param {Boolean} [dataConfig.noCache=true] - If true, attempt
+     *   to free page caches associated with the managed files
      * @param {werelogs.API} [logApi] - object providing a constructor function
      *                                for the Logger object
+     * @param {Boolean} [dataConfig.isPassthrough=false] - If true, enable
+     *   passthrough filesystem I/O
+     * @param {Boolean} [dataConfig.isReadOnly=false] - If true, disable
+     *   put and delete
      */
     constructor(dataConfig, logApi) {
         this.logger = new (logApi || werelogs).Logger('DataFileStore');
         this.dataPath = dataConfig.dataPath;
         this.noSync = dataConfig.noSync || false;
+        this.noCache = dataConfig.noCache || false;
```

Comment:
The JSDoc suggests the default value is `true`, here it seems `false` if not set.